Assets/Scripts/Models/Furniture.cs

```diff
@@ -143,6 +151,34 @@ private Furniture(Furniture other)
 
     public event Action<IPowerRelated> PowerValueChanged;
 
+    // PathfindingModifier is added to PathfindingCost to get the final amount
```

Comment:
Change these to documentation comments and they should show up in code completion.
```
/// <summary>
///  PathfindingModifier is added to PathfindingCost to get the final amount.
/// </summary>
```